output

```diff
@@ -0,0 +1,21 @@
+TERM_PROGRAM=Apple_Terminal
```

Comment:
Oh oops. That was created while I was developing. It isn't created each time; I'll delete it.